File: Sample/ContextApi.CS/ContextApi.Tests/MessageHelper.cs
using System;
using System.Linq;


namespace ContextApi.Tests
{
    public static class MessageHelper
    {
        public static string NoRecordsInDatabase(Type typeOfEntity)
        {
            string message = string.Format("No records of type {0} in the database!", typeOfEntity.Name);
            return message;
        }
    }
}

File: Sample/ContextApi.VB/ContextApi.Tests/MessageHelper.vb
Public NotInheritable Class MessageHelper
    Private Sub New()
    End Sub
    Public Shared Function NoRecordsInDatabase(ByVal typeOfEntity As Type) As String
        Dim message As String = String.Format("No records of type {0} in the database!", typeOfEntity.Name)
        Return message
    End Function
End Class
